geode-redis/src/main/java/org/apache/geode/redis/internal/executor/string/IncrByFloatExecutor.java

```diff
@@ -46,15 +44,14 @@ public RedisResponse executeCommand(Command command,
     byte[] incrArray = commandElems.get(INCREMENT_INDEX);
     String doub = Coder.bytesToString(incrArray).toLowerCase();
     if (doub.contains("inf") || doub.contains("nan")) {
```

Comment:
Redis uses the methods `isnan` (https://en.cppreference.com/w/c/numeric/math/isnan) and `isinf` (https://en.cppreference.com/w/c/numeric/math/isinf) from the math library, which seem to check more than the string literals.  Also, not all strings that contain `inf` or `nan` would produce NaN or Infinity.  Some are just invalid floats.

Reply:
Add a regex which should cover all options (I hope).